### DISCUSSION TITLE
How to customize the charts width and height in css

### DISCUSSION THREAD

**Discussion by di-coder** [apache/superset#15774] [Q&A / Help]
Hi Guys. I am new to the apache superset. I need to create a customized dashboard by changing the chart's position, width, and height. All charts have the same classes. So I am not able to change the specific chart CSS. If I change the position of the charts. All charts in the dashboard are getting affected.

**Comment by nytai** [comment]
You should be able to right click on the element and copy the charts css selector or use the id attr

**Reply by piewaer** [reply]
In the frontend ,My page will report an error, won't yours?
![image](https://user-images.githubusercontent.com/87263497/126146509-47fca5e4-ea9e-42aa-967d-ebf682675a3f.png)
